### PR TITLE
for loop format and var format

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
format for i := 0; i < 10; i++ {...}
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jianglingxia** [comment]
/assign @msau42
thanks

**Comment by jianglingxia** [comment]
PTAL @msau42
thanks

File: test/e2e/storage/vsphere_utils.go [hunk 1]
```diff
@@ -134,9 +134,7 @@ func getVSpherePersistentVolumeSpec(volumePath string, persistentVolumeReclaimPo
 
 // function to get vsphere persistent volume spec with given selector labels.
 func getVSpherePersistentVolumeClaimSpec(namespace string, labels map[string]string) *v1.PersistentVolumeClaim {
-	var (
-		pvc *v1.PersistentVolumeClaim
-	)
+	var pvc *v1.PersistentVolumeClaim
```
Comment (msau42) [inline test/e2e/storage/vsphere_utils.go]: Do you even need to declare it at the top?

**Pull request closed without merging**